IntegrationTests/Services/AWSSQSIntegrationTests/SQSTests.swift

```diff
@@ -17,7 +17,7 @@ class SQSTests: XCTestCase {
     private var queueUrl: String?
 
     override func setUp() async throws {
-        client = try await SQSClient()
+        client = try await SQSClient.builder().build()
```

Comment:
Is the noargs constructor no longer available?

Reply:
It's still available. I will try updating it to the following to verify the region is updated:
```
try await SQSClient.builder().withPlugin(RegionPlugin("us-west-1")).build()
```